.github/ISSUE_TEMPLATE/java-stable-release-template.md

```diff
@@ -0,0 +1,43 @@
+### RELEASE_VERSION_NUMBER-alpha.1 pre-release checklist 
+
+- [ ] Update the SNAPSHOT version in this repo's `README.md` file 
+- [ ] Change version name to `RELEASE_VERSION_NUMBER-alpha.1` in this repo's `gradle.properties` (make sure to remove `-SNAPSHOT`)
+- [ ] Create a new release in Github with a release title of `Mapbox Java SDK vRELEASE_VERSION_NUMBER-alpha.1` and a tag version of `vRELEASE_VERSION_NUMBER-alpha.1`. This will publish artifacts to Bintray.
+- [ ] Change version name back to having `-SNAPSHOT` at the end
+- [ ] Update this repo's `CHANGELOG.md`
+
+### RELEASE_VERSION_NUMBER-alpha.1 pre-release testing
+- [ ] Create a pull request in [`/mapbox-gl-native`](https://github.com/mapbox/mapbox-gl-native) that updates the Mapbox Maps SDK for Android's Java SDK dependency to `RELEASE_VERSION_NUMBER-alpha.1`
+- [ ] Create a pull request in [the Mapbox Navigation SDK for Android repo](https://github.com/mapbox/mapbox-navigation-android) that updates the Nav SDK's Java SDK dependency to `RELEASE_VERSION_NUMBER-alpha.1`
+- [ ] Report any regressions on this ticket
+
+### RELEASE_VERSION_NUMBER-alpha.2 pre-release checklist 
+- [ ] Change version name to `RELEASE_VERSION_NUMBER-alpha.2` in this repo's `gradle.properties` (make sure to remove `-SNAPSHOT`)
+- [ ] Change version name back to having `-SNAPSHOT` at the end
+- [ ] Update this repo's `CHANGELOG.md`
+- [ ] Create a new release in Github with a release title of `Mapbox Java SDK vRELEASE_VERSION_NUMBER-alpha.2` and a tag version of `vRELEASE_VERSION_NUMBER-alpha.2`
+
+### RELEASE_VERSION_NUMBER-alpha.2 pre-release testing
+- [ ] Create a pull request in [`/mapbox-gl-native`](https://github.com/mapbox/mapbox-gl-native) that updates the Mapbox Maps SDK for Android's Java SDK dependency to `RELEASE_VERSION_NUMBER-alpha.2`
+- [ ] Create a pull request in [the Mapbox Navigation SDK for Android repo](https://github.com/mapbox/mapbox-navigation-android) that updates the Nav SDK's Java SDK dependency to `RELEASE_VERSION_NUMBER-alpha.2`
+- [ ] Report any regressions on this ticket
+
+### Final RELEASE_VERSION_NUMBER release checklist 
+- [ ] Update this repo's `README.md` file 
+- [ ] Change version name to `RELEASE_VERSION_NUMBER` in `gradle.properties` (removing `-SNAPSHOT`)
+- [ ] Create a new release in Github with a release title of `Mapbox Java SDK vRELEASE_VERSION_NUMBER` and a tag version of `vRELEASE_VERSION_NUMBER`. This will publish artifacts to Bintray.
```

Comment:
This will push the artifact to Bintray, but it won't publish it automatically.

Reply:
Ok, have added a section about what to do in Bintray